### ISSUE TITLE
MS2 calibration (new) looks worse than (old) 

### ISSUE THREAD

**Issue by candidaorelmex** [Nesvilab/FragPipe#1866]
Hi,

I noticed that for my runs during "MASS CALIBRATION AND PARAMETER OPTIMIZATION" the (old)/measured mass offset is closer o 0 than the (new)/calibrated mass offset for MS2.

I was running a hybrid search (DIA and DDA files), the screenshot below is from the mass calibration of the DDA files.

![image](https://github.com/user-attachments/assets/84250a35-eb9f-4b04-8aa1-9995a978bcc3)

Is this WAI? Im processing immunopeptidomics data, a perhaps relevant detail. Please find the "fragpipe.workflow" in the attachments, is the reason for this in my settings?

Thanks for your help!

Best,
Ilja

[fragpipe.workflow.txt](https://github.com/user-attachments/files/17690958/fragpipe.workflow.txt)

**Comment by candidaorelmex** [comment]
Yes, that's what I meant.

Ok, so that it means there is no issue, if I understand correctly.
Thanks for the quick answer!

Best,
Ilja

**Comment by fcyu** [comment]
Yeah, as far as I can tell from the median and MAD, it is safe to say that there is no critical issues.

Best,

Fengchao
